WireAPI/Sources/WireAPI/APIs/ClientAPI/ClientAPIBuilder.swift

```diff
@@ -0,0 +1,59 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+/// A builder of `ClientAPI`.
+
+public struct ClientAPIBuilder {
+
+    let httpClient: any HTTPClient
+
+    /// Create a new builder.
+    ///
+    /// - Parameter httpClient: A http client.
+
+    public init(httpClient: any HTTPClient) {
```

Comment:
I thought as we go further we would use the APIService here, no?

Reply:
done in https://github.com/wireapp/wire-ios/pull/2019/commits/b4e5d87f6310bf53b4909ec3da5805e14fc79fed